### ISSUE TITLE
[keyboard] Pressing Command-[key] breaks key sequences & shortcuts

### ISSUE THREAD

**Issue by JacobZwang** [solidjs-community/solid-primitives#553]
### Describe the bug

When using `useKeyDownSequence` and `createShortcut` if you at any time use a shortcut such as Command-A, they will stop working as expected. This issue can be observed on the example provided in the keyboard package's readme.

1. Press Command + A.
2. Observe that the A stays in "Pressed Keys".
3. Press other keys and see they are added to the list with A. Notice the Control + E + R example does not work anymore.
4. Blur the window and notice the A is removed.

I've observed this behavior on MacOS Chrome and Safari, have not tried on Windows.

### Minimal Reproduction Link

https://codesandbox.io/s/solid-primitives-keyboard-demo-s2l84k?file=/index.tsx

**Comment by sid1029** [comment]
Agree. I have also observed this behavior. Trying to make copy paste working using Command + C, Command + V shortcuts is not working because of this bug.
Cmd + C works but a subsequent Cmd + V doesn't work until the browser window is blurred.

**Comment by blarfoon** [comment]
Can confirm this bug. Is anyone looking into it?

**Comment by thetarnav** [comment]
Not to my knowledge.
